packages/components/src/Inject/Inject.component.js

```diff
@@ -67,11 +67,11 @@ Inject.all = function injectAll(getComponent, components, CustomInject = Inject)
 	if (!getComponent || !components) {
 		return nothing;
 	}
-	return key => {
+	return (key, props) => {
```

Comment:
add props = {} as add default value

Reply:
No need too, spread of undefined works as expected